internal/controllers/clusterextension_controller.go

```diff
@@ -264,20 +267,15 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	unpackResult, err := r.Unpacker.Unpack(ctx, bundleSource)
 	if err != nil {
 		setStatusUnpackFailed(ext, err.Error())
+		setStatusProgressing(ext, err)
 		return ctrl.Result{}, err
 	}
 
 	switch unpackResult.State {
-	case rukpaksource.StatePending:
-		setStatusUnpackFailed(ext, unpackResult.Message)
-		ensureAllConditionsWithReason(ext, ocv1alpha1.ReasonFailed, "unpack pending")
-		return ctrl.Result{}, nil
 	case rukpaksource.StateUnpacked:
 		setStatusUnpacked(ext, unpackResult.Message)
```

Comment:
Should we setStatusProgressing here ? or just remove this completely (probably in the removed unpacked status pr) ?

Reply:
I would anticipate that we remove it completely in the unpacked status PR. There isn't really any value to setting it here since if there are no errors throughout reconciliation we set it to False, Succeeded